<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #a4b0ab;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>571</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="maze_tab">
     <attribute name="title">
      <string>MAZES</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>370</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="height">
         <property name="styleSheet">
          <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="width">
         <property name="styleSheet">
          <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="MazeWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>221</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="OpenFileButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color : #7f8f88;  
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #7f8f88, stop: 1 #5a7368);
}</string>
         </property>
         <property name="text">
          <string>OPEN FILE</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+O</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SaveButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color : #7f8f88;  
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #7f8f88, stop: 1 #5a7368);
}</string>
         </property>
         <property name="text">
          <string>SAVE TO FILE</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GeneataMazeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color : #7f8f88;  
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #7f8f88, stop: 1 #5a7368);
}</string>
         </property>
         <property name="text">
          <string>GENERATE MAZE</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+G</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="cave_tab">
     <attribute name="title">
      <string>CAVES</string>
     </attribute>
     <widget class="CaveWidget" name="cavewidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="GenerateCaveButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color : #7f8f88;  
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #7f8f88, stop: 1 #5a7368);
}</string>
      </property>
      <property name="text">
       <string>GENERATE NEW</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="height_input">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>180</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #4b5e56;</string>
      </property>
      <property name="text">
       <string>heigth</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="width_input">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>210</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>210</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #4b5e56;</string>
      </property>
      <property name="text">
       <string>width</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bc_input">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>240</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #4b5e56;</string>
      </property>
      <property name="text">
       <string>birth chance</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="die_if_less_input">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>270</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #4b5e56;</string>
      </property>
      <property name="text">
       <string>die if less</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>300</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #4b5e56;</string>
      </property>
      <property name="text">
       <string>born if more</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="born_if_more_input">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>300</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NextStepButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>340</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color : #7f8f88;  
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #7f8f88, stop: 1 #5a7368);
}</string>
      </property>
      <property name="text">
       <string>NEXT STEP</string>
      </property>
     </widget>
     <widget class="QToolButton" name="NextStepAutoButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>412</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-color : #7f8f88;  
	color: white;
	border: 1px solid gray;
}
QToolButton:checked{
    background-color: #4b5e56;
}</string>
      </property>
      <property name="text">
       <string>AUTOSTEP IN N MS</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="N_input">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>490</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #f2f5f4; color: #66736d;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>490</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #4b5e56;</string>
      </property>
      <property name="text">
       <string>step in N ms</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenCaveButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color : #7f8f88;  
	color: white;
	border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #7f8f88, stop: 1 #5a7368);
}</string>
      </property>
      <property name="text">
       <string>OPEN CAVE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CaveWidget</class>
   <extends>QWidget</extends>
   <header>cavewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MazeWidget</class>
   <extends>QWidget</extends>
   <header location="global">mazewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
